src/resources/views/pages/volumes/show.blade.php

```diff
@@ -18,7 +19,7 @@
                <h6 class="card-text">Release year<{{ $volume->release_year }}></h6>
            </div>
            <div class="card-footer text-muted">
-               2 days ago
+               x days ago
```

Comment:
This can be achived with: https://stackoverflow.com/questions/17260231/using-carbon-to-return-a-human-readable-datetime-difference